hub-chart/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Default values for KubeStellar-Hub.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+# Container/chart image versions. These values are set during the chart release process,
+# please do not change them unless you know what your are doing.
+HELM_VERSION: "3.14.0"
+KUBECTL_VERSION: "1.29.3"
+KUBESTELLAR_VERSION: "0.23.0-alpha.2"
+CLUSTERADM_VERSION: "0.8.2"
+OCM_STATUS_ADDON_VERSION: "0.2.0-rc8"
+OCM_TRANSPORT_PLUGIN_VERSION: "0.1.7"
+
+
+# Control controller log verbosity
+# The global VERBOSITY value will be sued for all controller unless a specific controller verbosity override is specified
+VERBOSITY: 2 # default global value for all controllers
+# Specific controller verbosity overrides:
+# KUBEFLEX_VERBOSITY_OVERRIDE: 6
+# KUBESTELLAR_VERBOSITY_OVERRIDE: 6
+# CLUSTERADM_VERBOSITY_OVERRIDE: 6
+# OCM_STATUS_ADDON_VERBOSITY_OVERRIDE: 6
+# OCM_TRANSPORT_PLUGIN_VERBOSITY_OVERRIDE: 6
```

Comment:
This is not used anywhere (due to the relevant Helm chart not having the needed parameter).

Reply:
I'm undecided... this was a way to provide an alternative to

```shell
helm upgrade --install add-wds2 oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION \
  --set='kubeflex-operator.install=false,InstallPCHs=false' \
  --set-json='WDSes=[{name":"wds2"}]'
```

as

```shell
helm upgrade --install add-wds2 oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION \
  --values=values-cp-only.yaml \
  --set-json='WDSes=[{name":"wds2"}]'
```

Reply:
I have implemented them pending the PRs, they will be ignored until then